composer.json

```diff
@@ -23,12 +23,12 @@
     ],
     "require": {
         "php": ">=8.1",
-        "phpcsstandards/phpcsextra": "^1.2.0",
-        "slevomat/coding-standard": "^8.16.0",
-        "squizlabs/php_codesniffer": "^3.11.3"
+        "phpcsstandards/phpcsextra": "dev-develop",
```

Comment:
FYI: The latest release(s) of PHPCSExtra is already fully compatible with PHPCS 4.0 (and cross-version compatible with 3.x too)